src/openforms/templates/includes/a11y_toolbar.html

```diff
@@ -1,11 +1,12 @@
 {% load i18n %}
 <nav class="a11y-toolbar">
-    <ul class="a11y-toolbar__list">
-        <li class="a11y-toolbar__list-item">
-            <a class="utrecht-link utrecht-link--html-a utrecht-link--openforms a11y-toolbar__window-print-action" href="#">{% trans "Print this page" %}</a>
-        </li>
-        <li class="a11y-toolbar__list-item">
-            <a class="utrecht-link utrecht-link--html-a utrecht-link--openforms" href="#main-content">{% trans "Back to top" %}</a>
-        </li>
-    </ul>
+    <hr class="openforms-backtotop-divider"/>
+    <p class="a11y-toolbar__list" role="group">
+        <a
+            class="utrecht-link utrecht-link--html-a utrecht-link--openforms openforms-backtotop-link"
+            href="#main-content">
+            <span aria-hidden="true" class="utrecht-icon"><i class="fa fas fa-icon fa-fw" aria-hidden="true"></i></span>
+            {% trans "Back to top" %}
+        </a>
+    </p>
```

Comment:
do we need this `p` tag at all?

I'm having doubts about `openforms-backtotop-divider` being a standalone component - in this form it's semantically not reusable at all because it's so specific. If it's so specific, I'd expect it to be a child of `a11y-toolbar` rather than a component by itself.